cmd/validator/credentials/set/process.go

```diff
@@ -192,7 +201,7 @@ func (c *command) generateOperationFromMnemonicAndValidator(ctx context.Context)
 			if c.debug {
 				fmt.Fprintf(os.Stderr, "Gone %d indices without finding the validator, not scanning any further\n", maxDistance)
 			}
-			break
+			return fmt.Errorf("failed to find validator using the provided mnemonic, validator=%s, pubkey=%#x.", c.validator, validatorInfo.Pubkey)
```

Comment:
The period needs to be removed here to satisfy the linter.

Reply:
Done. I also fixed the test output that some of the linting changed.